src/index.tsx

```diff
@@ -79,7 +79,6 @@ export default class InlineSVG extends React.PureComponent<IProps, IState> {
     this._isMounted = true;
 
     if (!canUseDOM()) {
-      this.handleError(new InlineSVGError('No DOM'));
```

Comment:
Why did you remove this? If a user didn't set a loader, it should warn him if there's nothing being rendered, right?

Reply:
This error implied that `react-inlinesvg` didn't support SSR environments. I don't think onError should be called in the pre-render stage.

Reply:
But is it a breaking change?
SSR is like a setup stage, and the client then calls either `onLoad` or `onError`

Reply:
Yes, if someone was watching this error to change the behavior...

Reply:
Supporting  SSR and throwing errors in it doesn't make much sense don't you think? In theory someone **might** be listening to that specific no DOM error, but as the error is called in `setState` function you can't really adjust the UI in SSR stage as only the first render happens there, at best you could log something to sentry or something like that. We have an SVG wrapper that swaps the output when an error occurs, but that doesn't work in SSRs pre-render stage.

Anyway what do you feel needs to be done here?

Reply:
Hey @gilbarbara , could you hint what you expect from this PR? So I could adjust accordingly.